extensions/sql-migration/src/wizard/accountsSelectionPage.ts

```diff
@@ -50,6 +50,8 @@ export class AccountsSelectionPage extends MigrationWizardPage {
 			if (value.selected) {
 				this.migrationStateModel.azureAccount = this.migrationStateModel.getAccount(value.index);
 				this.migrationStateModel.subscriptions = undefined!;
+				this.migrationStateModel._targetSubscription = undefined!;
```

Comment:
we should try to be consistent with the use of '_' prefix on variable names within a class (i.e. all fields should use them or now).  though '_' is typically meant to indicate a private variables in this codebase.